Makefile

```diff
@@ -19,14 +20,17 @@ KRUN=${KRUN_DIR}/krun.py
 KRUN_VERSION=master
 LIBKRUNTIME=${KRUN_DIR}/libkrun/libkruntime.so
 
+LIBKRUNPATH := ${realpath ${KRUN_DIR}/libkrun}
```

Comment:
Does that syntax for calling a command work? I think it might be a Gnu-ism? I think backquotes can be used instead? [I'm also not sure if we really need to call `realpath` or not!]

Reply:
I wanted to make sure this was simple variable expansion instead of recursive expansion (where realpath is called from the cd on each line). I blindly copied the GNU docs for that which use `${}`, but I guess I could try backticks instead.

Reply:
Do we even need `realpath`?

Reply:
Maybe not? `LIBKRUNPATH` gets passed to `ld` though and I've always considered it good practice to pass absolute paths where possible.

Reply:
My experience is that `realpath` is sometimes used to paper over problems so I'd be inclined to see if we can get away without it. [If we do need an absolute path, it's likely that the user should have passed us one in somewhere.]

Reply:
Ok, I'll give relative paths a go and see if it works.

Reply:
It seems that Rust does require an absolute library path. Personally I think for now using `realpath` in the Makefile is the least worst option.

Reply:
OK.

Reply:
I couldn't really find a nice way to do this which was both GNU and BSD make compatible, so I just use `realpath` where needed: f79256c967b98261bc66d0ee41a3f1003a08dea4